package-lock.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This is actually a file that's necessary - don't delete this file

Reply:
What's the reason for this file? I see `package.json` in the master branch but I don't see `package-lock.json`

Reply:
package-lock.json is similar to use commiting Gemfile.lock so that the dependencies are in sync everywhere

Reply:
Yeah somehow this didn't get included early on. Always want to check in the lock file.